virtio-devices/src/transport/pci_device.rs

```diff
@@ -798,9 +805,17 @@ impl VirtioPciDevice {
                 error!("Queue {} is not valid", queue_index);
             }
 
+            #[cfg(feature = "sev_snp")]
+            let access_platform = if self.sev_snp_enabled {
+                self.common_config.access_platform.clone()
```

Comment:
Could you simplify this and always just clone the access_platform (since cloning `None` gives you `None`).